website/docs/layouts/grid/partials/code/how-to-use.md

```diff
@@ -0,0 +1,299 @@
+## How to use this component
+
+!!! Insight
+
+While it’s not necessary to be familiar with CSS3 grid specifications to use this component, some knowledge may be helpful in achieving more complex layouts.
+
+A helpful reference with clear examples: [CSS Tricks: Complete grid layout guide](https://css-tricks.com/snippets/css/complete-guide-grid/).
+
+!!!
+
+The `Layout::Grid` and optional `Layout::Grid::Item` components provide a way to quickly build out flexible grid-based layouts of components or elements without needing to write a lot of custom CSS code or understand all the intricacies of CSS grid styles.
+
+### Basic usage
+
+The parent `Grid` component creates a layout of equal width “columns” by default. Other components and HTML elements can be passed in directly.
+
+```handlebars
+<Hds::Layout::Grid class="doc-grid-outline-children">
+  <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 2" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
```

Comment:
[suggestion] Since we are already declaring the background color inline in the snippet, what about using different colors to differentiate the items (with the benefit of making outlining them not necessary anymore? you may want to leave the outline only on the grid container, in case)

```suggestion
  <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4c5f3" />
  <Doc::Placeholder @height="40px" @text="Item 2" @background="#e5ffd2" />
  <Doc::Placeholder @height="40px" @text="Item 3" @background="#d2f4ff" />
  <Doc::Placeholder @height="40px" @text="Item 4" @background="#fff8d2" />
```

Reply:
I'm updating all these locally.